app/assets/stylesheets/application.scss

```diff
@@ -8,6 +8,7 @@
 @import "govuk_publishing_components/components/error-alert";
 @import "govuk_publishing_components/components/error-message";
 @import "govuk_publishing_components/components/error-summary";
+@import "govuk_publishing_components/components/govspeak";
```

Comment:
I couldn't find an instance of govspeak being used, is it nested in another component or something?

Reply:
Yes, it's nested in the `notice` component. We probably don't use that option of the component but it's worth having in place in case we suddenly start.